### ISSUE TITLE
SecurityGroups: list of security groups should contain shared SGs.

### ISSUE THREAD

**Issue by velp** [sapcc/elektra#929]
Elektra does not show security groups shared via neutron RBAC:

For example, we have shared SG with UUID `b5c53c61-8bc5-43e3-9832-3d019f1cd004`
```
 $ os security group list
+--------------------------------------+-------------------------------------+-------------------------------------+----------------------------------+----------+
| ID                                   | Name                                | Description                         | Project                          | Tags     |
+--------------------------------------+-------------------------------------+-------------------------------------+----------------------------------+----------+
| 1a5b0497-010e-469d-8b3a-7aa9e19df07b | d063079-test-sg                     |                                     | e9141fb24eee4b3e9f25ae69cda31132 | []       |
| 848e8ac6-9f2a-4286-9bab-6a61d7f0c1d7 | d072895-test-sg                     | my test group                       | e9141fb24eee4b3e9f25ae69cda31132 | []       |
| b5c53c61-8bc5-43e3-9832-3d019f1cd004 | test-sg-rbac                        |                                     | 7243f3f80a99434292d6766069d3961a | []       |
+--------------------------------------+-------------------------------------+-------------------------------------+----------------------------------+----------+
```
but we cannot find it in dashboard:
![image](https://user-images.githubusercontent.com/8326634/131354730-a6445fd7-9ea9-4c3d-a7da-155b42aaddfb.png)

**Comment by andypf** [comment]
@velp what is the API endpoint?

**Comment by andypf** [comment]
At the moment we are using the endpoint from the compute: "servers / :server_id / os-security-groups"

**Comment by velp** [comment]
You have to use the same endpoint but without tenant_id/project_id filtering because shared resources will have another project.

The example of a response:

```
$ curl -s -X GET -H "User-Agent: openstacksdk/0.57.0 keystoneauth1/4.3.1 python-requests/2.25.1 CPython/3.9.5" -H "X-Auth-Token: $(os token issue -c id -f value)" "https://network-3.qa-de-1.cloud.sap/v2.0/security-groups?fields=id&fields=name&fields=description&fields=project_id&fields=tags" | jq .
{
  "security_groups": [
    {
      "id": "1a5b0497-010e-469d-8b3a-7aa9e19df07b",
      "name": "d063079-test-sg",
      "shared": true,
      "description": "",
      "tags": [],
      "project_id": "e9141fb24eee4b3e9f25ae69cda31132"
    },
    {
      "id": "5fd2a794-e6a6-4086-8d62-5e4271d65c22",
      "name": "default",
      "shared": true,
      "description": "Default security group",
      "tags": [
        "test"
      ],
      "project_id": "e9141fb24eee4b3e9f25ae69cda31132"
    },
    {
      "id": "b5c53c61-8bc5-43e3-9832-3d019f1cd004",
      "name": "test-sg-rbac",
      "shared": true,
      "description": "",
      "tags": [],
      "project_id": "7243f3f80a99434292d6766069d3961a"
    }
  ],
  "security_groups_links": [
    {
      "rel": "previous",
      "href": "https://network-3.qa-de-1.cloud.sap/v2.0/security-groups?fields=id&fields=name&fields=description&fields=project_id&fields=tags&marker=06d74a24-d305-423b-a0cc-f4c6ea69e6f1&page_reverse=True"
    }
  ]
}
```

**Comment by velp** [comment]
All security groups will be returned in one call (without `project_id` filtering). Neutron doesn't have a special endpoint for shared resources. If we cannot get all security groups in one list for some reason (I example sent above) - then let's do it in two calls.

**Comment by andypf** [comment]
fixed with 68fc1b1895e9af23f6a033b095e1ec6dda5204b5
